src/widgets/mainwindow.h

```diff
@@ -93,6 +95,7 @@ private slots:
     bool             _useSIPrefixes;
     QTimer           _purgeTimer;
     int              _purgeTimerCount;
+    QMessageBox      *_nukeCountdownWindow;
```

Comment:
It's used by two different methods.  If we don't keep a pointer, I don't think we can adjust the countdown text.